### DISCUSSION TITLE
Working on a nightly RPM build of Neovide

### DISCUSSION THREAD

**Discussion by chrisbouchard** [neovide/neovide#513] [Show and tell]
I wanted to let you know I'm working on a Fedora COPR (community project) to build Neovide nightly as an RPM, mostly so I can more easily install it on all my computers. It's not actually running nightly yet, as I'm still working the kinks out.

The COPR is here: https://copr.fedorainfracloud.org/coprs/chrisbouchard/neovide-nightly/
And the build configuration is here: https://github.com/chrisbouchard/neovide-nightly

I actually had one question as I work on packaging. Is the `assets/nvim.ico` icon in the repository intended to be the icon for the desktop entry? I notice the desktop entry has `Icon: neovide`. I can easily enough install the icon file with the altered name, but I wanted to understand your intentions.

Thanks again for an awesome editor.

**Comment by chrisbouchard** [comment]
Per [this other discussion](/Kethku/neovide/discussions/515) and [!519](/Kethku/neovide/pull/519), I'm now using the `assets/neovide.svg` icon packaged with Neovide.

I've finally set up the nightly trigger, so my first automated RPM build of Neovide should be available tomorrow. There are already several that I triggered manually, and I'm using RPM nightly Neovide as my daily-driver neovim editor.

**Reply by Kethku** [reply]
Very cool. Is there anything I should add to the readme to indicate this? I don't know what one needs to know to use the package in fedora

**Reply by chrisbouchard** [reply]
All the instructions _should_ be in the [chrisbouchard/neovide-nightly](https://github.com/chrisbouchard/neovide-nightly) readme. Coprs are pretty easy to use in Fedora. Though I have to beef my readme up a bit to explain exactly what is being built and when.

**Reply by ratmav** [reply]
ya'll want a pr for the readme update? i mean it wasn't hard to find this discussion using the github search, but a heads up on the readme would have been nice. i was about to start a source build and went looking for a list of red hat package names for the deps and found this first.

**Reply by j4qfrost** [reply]
Have you run into any fontconfig issues? That seems like a popular problem surrounding packaging.

**Reply by ratmav** [reply]
just installed it. i mean so far, so good. font seems normal.

**Reply by chrisbouchard** [reply]
@j4qfrost The RPM [depends on `fontconfig` (among others)][rpm-deps], so as long as the latest version is OK it should be good. I haven't had any font-related issues<sup id="ref-1">1</sup> in my personal use.

@ratmav FWIW, I definitely didn't mean that we _shouldn't_ update the readme (though I see now how it could be read that way), just that this project should feel free to copy anything I've written in my RPM project's readme. I know this reply's a bit late; if you still want to make the PR, I'd say go for it, otherwise I can _soon_&trade; when I get some time.

---

<sup>1</sup>  Aside from #213, so the RPM also depends on `google-noto-sans-mono-fonts`. I haven't tried relaxing that dependency in a while, so I don't know if it's still a requirement.

[rpm-deps]: https://github.com/chrisbouchard/neovide-nightly/blob/main/neovide.spec#L26-L52
